src/engines/html5/html5.js

```diff
@@ -240,7 +252,35 @@ export default class Html5 extends FakeEventTarget implements IEngine {
   get id(): string {
     return Html5.id;
   }
-
+  /**
+   * Listen to the video element events and triggers them from the engine.
+   * @public
+   * @param {boolean} stopSeeking don't seek to the last detach point
+   * @returns {void}
+   */
+  attachMediaSource(stopSeeking: ?boolean): void {
+    if (!this._mediaSourceAdapter) {
+      this.reset();
+      this._init(this._source, this._config);
+      if (!isNaN(this._lastTimeDetach) && !stopSeeking) {
```

Comment:
just replace `stopSeeking` with `this.ended` and you can remove the parameter here in the player

Reply:
@Yuvalke did you check this feature with bumper with/out ima?

Reply:
No, Because mi droll could appear on the end after seek and the other ads on the will not appear

Reply:
If I'll not seek the next ad will not appear otherwise l don't want to seek to the end of all ads and content because I want to make sure that playback ended will appear once on the end of everything and reply button will appear(seek after every thing finished will hide it)